tests/integration/test_charm.py

```diff
@@ -47,3 +48,20 @@ async def test_waits_for_config_server(ops_test: OpsTest) -> None:
         status="Missing relation to config-server.",
         timeout=300,
     )
+
+
+@pytest.mark.group(1)
+@pytest.mark.abort_on_fail
+async def test_mongos_external_connections(ops_test: OpsTest) -> None:
+    """Tests that mongos is accessible externally."""
+    assert_node_port_available(ops_test, node_port_name="mongos-k8s-nodeport")
+
+    # TODO add this in once DPE-5040 / PR #20 merges
```

Comment:
I tested this on a fork of `DPE-5040` and it worked great, but I didn't want to have to block this PR while #20 merges (given Neha is OOO today and Pedro is OOO next week)